scripts/mk.js

```diff
@@ -3358,16 +3384,16 @@ function startGame() {
 			oPlayer.team = aTeams[i];
 		else
 			oPlayer.team = -1;
-		if (course != "BB") {
+		if (!isBattle()) {
 			posKart(oPlayer, isTT ? 1:oPlace);
 			oPlayer.time = 0;
-			oPlayer.tours = 1;
-			oPlayer.demitours = cp0;
-			//oPlayer.tours = oMap.tours;
-			//oPlayer.demitours = cp0 ? oMap.checkpoint.length-2:oMap.checkpoint.length-1;
-			//oPlayer.arme = "bloops";
-			//oPlayer.roulette = 24;
-			oPlayer.billball = 0;
+			oPlayer.laps = 1;
+			oPlayer.uturns = cp0;
```

Comment:
Not sure of this rename to `uturns` it's not exactly what it represents (even in french the name is quite not clear... it's demitours as "half laps", but technically it's more "part of laps").
In practice this variable contains the number of checkpoints you passed in the current lap. In the end "checkpoints" would likely be a better name.